docsrc/imap/reference/manpages/systemcommands/cyr_withlock_run.rst

```diff
@@ -0,0 +1,77 @@
+.. cyrusman:: cyr_withlock_run(8)
+
+.. author: Bron Gondwana
+
+.. _imap-reference-manpages-systemcommands-cyr_withlock_run:
+
+================
+**cyr_withlock_run**
+================
```

Comment:
Sphinx will complain about the overline and underline not being the same length as the heading

```suggestion
====================
**cyr_withlock_run**
====================
```